util/reggen/reg_block.py

```diff
@@ -584,6 +584,12 @@ def apply_alias(self, alias_block: 'RegBlock', where: str) -> None:
             raise ValueError('Alias register names {} are not unique in alias '
                              ' {}'.format(list(intersection), where))
 
+        # Build a name map for regwens so that we can check correspondence.
+        alias_wennames = {}
```

Comment:
ah nice. do you think i could eventually re-use this structure here for the regwen checks? (ie, a regwen is used, and that a referenced regwen actually exists?)

Reply:
you can probably use the same construction, yes. 
although it is temporary at the moment. 
we can add the map to the class if needed (like the other name maps).